docs/ref/forms/api.txt

```diff
@@ -1003,23 +1005,45 @@ method you're using:
     ...     "sender": "invalid email address",
     ...     "cc_myself": True,
     ... }
-    >>> f = ContactForm(data, auto_id=False)
-    >>> print(f)
-    <div>Subject:
-      <ul class="errorlist"><li>This field is required.</li></ul>
-      <input type="text" name="subject" maxlength="100" required aria-invalid="true">
+    >>> f = ContactForm(data)
```

Comment:
```suggestion
    >>> ContactForm(data)
```

Reply:
Good spot! I needed to either print it or add `.as_div()`. Gone for `.as_div()`